### DISCUSSION TITLE
Control of battery profile and Power Flow diagram

### DISCUSSION THREAD

**Discussion by firo** [binsentsu/home-assistant-solaredge-modbus#158] [Q&A]
Ciao! First of all thanks for sharing this HA integration with SolarEdge and thank you in advance for any reply.

I installed successfully the integration and for months all is fine. Yesterday I tried to change the control of battery profile for  charging from PV and AC for a test, following the suggestions in the homepage (Appendix B etc). 

All worked fine and I restored all original settings, however my power flow diagram disappeared from mySolaredge mobile app and monitor website. BTW all seems working fine and I run all test for the inverter status (ok) and connection status (ok).

This is few days ago screenshot
![image](https://user-images.githubusercontent.com/471014/235525442-aefd4e68-2968-4bc9-88e9-f59403be8ccc.jpeg)

And this is today 🤔 without inverter, battery, home consumption and so on
![image](https://user-images.githubusercontent.com/471014/235525545-6cf9bd01-e958-4595-a4d7-627c83cf88f2.jpeg)

Does anyone have any suggestions to restore the power flow diagram?

Thank you!
Firo

**Comment by jovana** [comment]
Hi, I have the same issue. Probably it has something to do with changing control settings. BTW I'm using the more up-to-date version located here: https://github.com/WillCodeForCats/solaredge-modbus-multi/discussions/296

Installation is working fine, but the Solaredge app does not seem to show the battery anymore.

**Reply by jovana** [reply]
I have an update on this. It seems like on some settings you made in HA the SE app is confusing about what and how to display. 

So change settings will change also the view.
